src/stress/texture/large.spec.ts

```diff
@@ -0,0 +1,24 @@
+export const description = `
+Stress tests covering usage of very large textures.
+`;
+
+import { makeTestGroup } from '../../common/framework/test_group.js';
+import { GPUTest } from '../../webgpu/gpu_test.js';
+
+export const g = makeTestGroup(GPUTest);
+
+g.test('loading')
+  .desc(
+    `Tests execution of shaders loading values from very large (at least 8192x8192)
+textures. The texture size is selected according to the limit supported by the
+GPUDevice.`
```

Comment:
Could potentially be expanded to very large 2d array or 3d textures as well (though they might run out of memory).

The base limit for 2d textures is 8192x8192 with 2048 layers.
The base limit for 3d textures is 2048x2048x2048.

Reply:
Done, added sub cases for 2d, 2d_array, and 3d